content/zh/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -57,7 +57,7 @@ kubectl run --image=nginx nginx-app --port=80 --env="DOMAIN=cluster"
 -->
 ```shell
 # 启动运行 nginx 的 Pod
-kubectl run --image=nginx nginx-app --port=80 --env="DOMAIN=cluster"
+kubectl create deployment --image=nginx nginx-app --port=80 --env="DOMAIN=cluster"
```

Comment:
do you mean we can specify `--port` in `kubectl` 1.18?

Reply:
Sorry, I didn't remove /hold after submitting the changes. Thank you for your timely review. The PR update has been completed.reference this [PR](https://github.com/kubernetes/website/pull/21536).